src/index.ts

```diff
@@ -881,6 +942,20 @@ class NotificationAPIClient implements NotificationAPIClientInterface {
       return;
     }
 
+    // create and insert the message at the top only if askForWebPushPermission is true
+    if (askForWebPushPermission) {
+      const message = document.createElement('div');
+      message.innerHTML =
```

Comment:
I'm not 100% if I remember our conversation, but I think we decided to have 2 opt-in:

One inside the in-app notifications at the top, but if they ignore that, then it will disappear.
One inside the user preferences (the one you have implemented). 

Is that correct?

Reply:
Please give me a design for in app one.